app/models/line_item.rb

```diff
@@ -1,5 +1,10 @@
 class LineItem < ApplicationRecord
-    belongs_to :product
-    belongs_to :cart
-    belongs_to :order,optional: true
-end
+  belongs_to :product
+  belongs_to :cart
+  belongs_to :order,optional: true
+  
+  def total_price
+    self.quantity ||= 0
+    self.quantity * self.product.price
```

Comment:
@amitshethe Here self is not required at Line 8.

Reply:
I tried without self keyword it doesn't show the current quantity